src/classes/GE_PaymentServices_TEST.cls

```diff
@@ -52,19 +52,19 @@ public class GE_PaymentServices_TEST {
 
     // Key-Values by service name
     private static final Map<String, List<Map<String, String>>> testConfig =
```

Comment:
I'm wondering if this test config would be better being defined in the PS_IntegrationServiceConfig test class? And maybe some of these test methods (shouldUsePaymentServicesConfig) as well?

Reply:
Done.